extensions/skins/conversation_v1/player.html

```diff
@@ -2,107 +2,258 @@
+
+    <div class="conversation-skin-oppia-feedback">
+      <div ng-if="feedback() === null">
+        <div class="conversation-skin-feedback-dot-one"></div>
+        <div class="conversation-skin-feedback-dot-two"></div>
+        <div class="conversation-skin-feedback-dot-three"></div>
       </div>
+      <div ng-if="feedback() !== null" angular-html-bind="feedback()"
+           class="protractor-test-conversation-feedback"></div>
     </div>
   </div>
 </script>
+
+<script type="text/ng-template" id="components/progressDots">
+  <style>
+    @keyframes oppia-animate-enter-progress-dot {
+      0% {
+        margin-left: -10px;
+      }
+
+      100% {
+        margin-left: 0;
+      }
+    }
+
+    .oppia-progress-arrow {
+      color: #ccc;
+      display: inline-block;
+      margin-left: 10px;
+      vertical-align: middle;
+      -webkit-user-select: none;
+      user-select: none;
+    }
+    .oppia-progress-arrow-active {
+      color: #fff;
+      cursor: pointer;
+    }
+
+    .oppia-progress-dot {
+      border-radius: 50%;
+      float: left;
+      height: 12px;
+      margin-left: 10px;
+      width: 12px;
+    }
+    .oppia-progress-dot-active {
+      background-color: #fff;
+    }
+    .oppia-progress-dot-inactive {
+      background-color: #ccc;
+      cursor: pointer;
+    }
+
+    .oppia-progress-dot-list {
+      list-style-type: none;
+      margin: 10px auto;
+      padding-left: 0;
+      position: relative;
+      text-align: center;
+    }
+
+    .oppia-animate-progress-dot {
+      display: inline-block;
+      vertical-align: middle;
+    }
+    .oppia-animate-progress-dot.ng-enter {
+      -webkit-animation: 400ms oppia-animate-enter-progress-dot;
+      animation: 400ms oppia-animate-enter-progress-dot;
+      -webkit-animation-timing-function: linear;
+      animation-timing-function: linear;
+    }
+  </style>
+
+  <ul class="oppia-progress-dot-list">
+    <li class="oppia-progress-arrow" ng-show="dots.length > 1"
+        ng-class="{'oppia-progress-arrow-active' : currentDotIndex > 0}"
+        ng-click="decrementCurrentDotIndex()">
+      <span class=" glyphicon glyphicon-arrow-left">
+      </span>
+    </li>
+    <li ng-repeat="dot in dots track by $index" class="oppia-animate-progress-dot">
+      <span class="oppia-progress-dot oppia-progress-dot-active"
+            ng-if="$index === currentDotIndex" ng-show="dots.length > 1">
+      </span>
+      <span class="oppia-progress-dot oppia-progress-dot-inactive"
+            ng-if="$index !== currentDotIndex"
+            ng-click="changeActiveDot($index)">
+      </span>
+    </li>
+    <li class="oppia-progress-arrow" ng-show="dots.length > 1"
+        ng-class="{'oppia-progress-arrow-active' : currentDotIndex != dots.length - 1}"
+        ng-click="incrementCurrentDotIndex()">
+      <span class=" glyphicon glyphicon-arrow-right">
+      </span>
+    </li>
+  </ul>
+</script>
+
+<script type="text/ng-template" id="popover/answer">
```

Comment:
Yes, I think so. It's a quirk with UI bootstrap popovers, and follows the pattern for the feedback popover.